HTML & CSS Practice

martthaa opened the pull request:
### HTML & CSS Practice

[Demo](https://martthaa.github.io/sweetPopUp/) |
[Code base](https://github.com/martthaa/sweetPopUp)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

kasionio commented:
Hi Martha. I didn't find any files in PR, please check

martthaa commented:
Hi @kasionio, in submissions folder? In PR I added a link [Code Base](https://github.com/martthaa/sweetPopUp) to the repository with the code.

kasionio commented:
> Hi @kasionio, in submissions folder? In PR I added a link [Code Base](https://github.com/martthaa/sweetPopUp) to the repository with the code.

You should add files that need to be reviewed 
https://kottans.org/documentation/docs/doc/contribute

kasionio commented:
Still nothing 🤷🏼‍♂️

martthaa commented:
привіт, @kasionio ) перевірте, будь ласка, чи все гаразд. Я старалась розібратись чому в мене в репозиторії була сіра папка з білою стрілочкою...і коли я натискала на папку, вона не відкривалася, але на моїй локальній машині все було гаразд, вміст був, але в github я не могла отримати до нього доступ.  Як я пізніше дізналась це - папка (локально) є вкладеним Git-репозиторієм , дерево якого SHA1 записується як «gitlink» (сіра папка з прямою білою стрілкою). Я довго пробувала вирішити це, але не вийшло( тому спробувала додати фали через інтерфейс git...ось посилання на мою [папку](https://github.com/martthaa/frontend-2022-homeworks/tree/html_css_Practice/submissions/martthaa/popUp)

kasionio commented:
Let's also practice English :) Stackoverflow is always ready to help you. First, read it carefully and decide how you can solve it. If you'll still have questions, ask here
https://stackoverflow.com/questions/62056294/github-folders-have-a-white-arrow-on-them

martthaa commented:
ok) thanks for the advice

martthaa commented:
but you will review these files? Or I should I try it again via git ?

kasionio commented:
> but you will review these files? Or I should I try it again via git ?

You should fix it anyway, and I'll look at your PR anyway also :)

kasionio reviewed (changes requested):
Great job, cool design, I really like it! 
Please take a look at the comments and screenshots. And try to use a keyboard, there is no focus on the popup button. Also, try to improve naming, `box`, `last`, `more` is not good because sounds too general, don't be afraid to use more than word :) You can read [this](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/) article, but don't take it too seriously, it is just one point of view.
![image](https://user-images.githubusercontent.com/21179601/186744464-5defccc1-0761-458d-beaf-77332fbf15b4.png)
![image](https://user-images.githubusercontent.com/21179601/186744572-8562ee4e-9113-48ef-ae88-a100e757992c.png)

kasionio commented:
And try to resolve issue with git, it is mandatory

martthaa commented:
I realized my mistake, when I added files, I simply copied the folder that I had previously added to to git repository. As a result, I seemed to want to create a repository in the middle of the repository, which is not correct. Today I already added a new project DOM and it's actually a piece of cake to add folder to an existing repository, I don't know why it was so difficult for me last time. Thanks for the corrections, tips and advice. I will try to fix everything tomorrow

kasionio commented:
No need to hurry, take your time. Anyway, I'll check it on Monday I guess

martthaa commented:
Done : )

kasionio commented:
> 

Take a look also in mobile view, now it looks not usable :(
![Screenshot_2022-08-26-22-13-37-641_com android chrome](https://user-images.githubusercontent.com/21179601/186975876-0e73926b-8378-4037-b113-8f4da7a2b618.jpg)

martthaa commented:
[DEMO](https://martthaa.github.io/sweetPopUp/)

kasionio commented:
Once you are done with the changes, please re-request review
![image](https://user-images.githubusercontent.com/21179601/187194728-077a4ea0-c9aa-4673-9e9c-4c2f49a9251e.png)

martthaa commented:
Make any other changes? or just click re-request review?

kasionio commented:
I can't reach the popup from the keyboard now, as I don't see an active state on it. Also seems that some style fixes needed 
![image](https://user-images.githubusercontent.com/21179601/187200217-fcf72924-f08e-47c4-95a3-ba6f3c5e7ba9.png)

martthaa commented:
I hope everything will be ok, [Demo](https://martthaa.github.io/sweetPopUp/)

kasionio commented on submissions/martthaa/popUp/index.html:
```diff
@@ -0,0 +1,125 @@
+    <header>
+      <nav class="nav">
+        <ul class="boxOfNavigationButtons">
+          <li class="list-item icon-recipes" tabindex="1">
```
tabindex is not needed within this task

kasionio commented on submissions/martthaa/popUp/style.css:
```diff
@@ -0,0 +1,214 @@
+  font-family: 'Brush Script MT', cursive;
+}
+
+.innerText{
```
still not implemented

kasionio commented on submissions/martthaa/popUp/style.css:
```diff
@@ -0,0 +1,225 @@
+}
+
+a:hover {
+  color: 808080;
```
missed `#`?

kasionio commented on submissions/martthaa/popUp/index.html:
```diff
@@ -0,0 +1,125 @@
+  <body>
+    <header>
+      <nav class="nav">
+        <ul class="boxOfNavigationButtons">
```
why camelCase?

kasionio reviewed (changes requested):
please look at the padding 
![image](https://user-images.githubusercontent.com/21179601/187279546-48b512da-5335-43e5-8dfd-6823ca2e1a41.png)

martthaa commented:
Hi) about span:
 I didn't delete it because I use it to give proper text size. When I tried to give a font-size in parent class it causeв to photo resizing also. I tried to use :not(.button-icon) for parent class but it didn't help so for now

martthaa commented:
[Demo](https://martthaa.github.io/sweetPopUp/)

kasionio reviewed (changes requested):
1. Containers on links jumping on hover
2. No pointer on hover on the popup button
3. Clicking the tab at the keyboard leads to the link at the bottom, which is not expected behavior 
4. `More` button has no focus state
5. Weird focus on link, looks below 

![image](https://user-images.githubusercontent.com/21179601/188327836-73564150-f93a-4b62-9edf-27c6180a4310.png)

martthaa commented:
[Demo](https://martthaa.github.io/sweetPopUp/)

kasionio commented:
> Hi) about span: I didn't delete it because I use it to give proper text size. When I tried to give a font-size in parent class it causeв to photo resizing also. I tried to use :not(.button-icon) for parent class but it didn't help so for now

You need to understand what `em` unit means. Please try to read any article about it and you will be able to remove it :) Also, `rem` is widely used today, read about it

kasionio commented on submissions/martthaa/popUp/style.css:
```diff
@@ -0,0 +1,232 @@
+
+@media screen and (max-width:540px) {
+  .popup{
+    right: -240%;
```
why percents here and px in other @media below at the same time?

kasionio replied:
with your current design seems you don't need to change popup right and top

kasionio commented on submissions/martthaa/popUp/style.css:
```diff
@@ -0,0 +1,232 @@
+  }
+}
+
+
```
too many empty lines :)

martthaa commented:
[DEMO](https://martthaa.github.io/sweetPopUp/)

kasionio commented on submissions/martthaa/popUp/style.css:
```diff
@@ -0,0 +1,224 @@
+  }
+}
+
+@media screen and (max-width:375px) {
```
why do you need so many breakpoints? it is a good practice to use several common breakpoints
https://www.lambdatest.com/blog/how-to-use-css-breakpoints-for-responsive-design/

kasionio reviewed (changes requested):
I don't see a focus state on `More` button when using a keyboard

martthaa commented:
> I don't see a focus state on `More` button when using a keyboard
![image](https://user-images.githubusercontent.com/77829665/189182755-0289317d-e1cf-4c95-8944-5b490ad07a18.png)

kasionio reviewed (changes requested):
Please try to click Space to open an additional menu, nothing happens

martthaa commented:
> Please try to click Space to open an additional menu, nothing happens
actually, I don't understand why it doesn't work (
![image](https://user-images.githubusercontent.com/77829665/189410010-6287ea2f-a1b5-46d3-a9c9-74ef886df8d9.png)

kasionio commented:
> > Please try to click Space to open an additional menu, nothing happens
> > actually, I don't understand why it doesn't work (
> > ![image](https://user-images.githubusercontent.com/77829665/189410010-6287ea2f-a1b5-46d3-a9c9-74ef886df8d9.png)

Please try to look through HTML and dev console and try to understand how your code works
